### ISSUE TITLE
How to update brunch?

### ISSUE THREAD

**Issue by Farhan811360** [sebanc/brunch#2048]
Hello! I'm going to install chrome os but I have question. How to update the brunch after I successfully installed chrome os?

**Comment by TrisoPcoo** [comment]
Correct me if I'm wrong, but I believe it's the [normal method (Google Support)](https://support.google.com/chromebook/answer/177889?hl=en-GB).

**Comment by Farhan811360** [comment]
Yes i know it, but in the brunch file note it was written "Update kernels to r123." Does it mean I have to update brunch file before I go to r124 update? If it is how do I update it?
